test/e2e/crdvalidator_test.go

```diff
@@ -2,15 +2,19 @@ package e2e
 
 import (
 	"context"
+	"fmt"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	rukpakv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
```

Comment:
supernit: any of the rukpak related packages should live in the last section of the package imports.